src/ex1/script.js

```diff
@@ -89,13 +91,13 @@ function filterTodo(event) {
 }
 
 function clearAllTasks() {
-    const allTodos = document.querySelectorAll('.todo')
-    allTodos.forEach(task => {
+    const allTodos = document.querySelectorAll(".todo");
+    allTodos.forEach((task) => {
         task.classList.add("fall");
-        task.addEventListener("transitionend", function () {
+        setTimeout(() => {
             task.remove();
-        });
-    })
+        },500);    
```

Comment:
where this magic comes from? is it used anywhere else? don't you need it in line 57 as well? consider exporting to a constant

Reply:
I can use this delay in line 57.
The tutorial I follow along to showed this 'transitionend' event listener which was nice to learn.
I used setTimeout in line 97 because of bug(2) which did not delete the 'fade-out' task. the event listener did not catch the 'transitionend' event so I add a delay instead for the task.remove() this way the animation can be done and just then the item will be deleted.